app/src/main/java/org/simple/clinic/scheduleappointment/ScheduleAppointmentEvent.kt

```diff
@@ -46,3 +46,7 @@ object SchedulingSkipped : ScheduleAppointmentEvent() {
 
 data class TeleconsultRecordLoaded(val teleconsultRecord: TeleconsultRecord?) : ScheduleAppointmentEvent()
 
+object OpenTeleconsultStatusSheet : ScheduleAppointmentEvent() {
```

Comment:
Let's rename `OpenTeleconsultStatusSheet` to `NextButtonClicked`, because we don't know if we will always open teleconsult status sheet on this button click and also we usually name the UI event names based on the action of the UI element.